isso/config.py

```diff
@@ -107,7 +125,7 @@ def section(self, section):
 
 def new(options=None):
 
-    cp = IssoParser(allow_no_value=True)
+    cp = IssoParser()
```

Comment:
Note that I moved the `allow_no_value` arg into the `__init__` function to make it easier to keep things in sync. Otherwise you'd have to adjust the tests as well every time `IssoParser` needs new args.